_pages/about/index.html

```diff
@@ -8,15 +8,39 @@
   <h1>{{{ iconmonstr 'about' }}}About</h1>
   <section class="module about">
     <p>
-      Async meets monthly in Brighton & Hove, UK, to learn, discuss and play with JavaScript and related web technologies.
+      Async is an inclusive Web Tech Meetup based in Brighton. Since 2010 we have been the place for people in and around Brighton who are interested in web technologies to learn, discuss, and play.
```

Comment:
```suggestion
      Async is an inclusive web tech meetup based in Brighton. Since 2010 we have been the place for people in and around Brighton who are interested in web technologies to learn, discuss, and play with web technologies.
```